docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -366,6 +370,19 @@ $ tctl create -f reviewer.yaml
 The `requester` will now only be able to search for or request access to
 resources accessible by the `staging-access` role.
 
+### Allow reviewers to see the hostname of nodes they do not normally have access to
+
+It is possible for a reviewer to view Resource Access Requests for SSH nodes to
+which the reviewer themself does not have access.
+In this case they will not be able to view the hostname of the requested node
+and they will only see the node's UUID.
+To give the reviewer permission to view the node hostname, use the
+`allow.review_requests.preview_as_roles` field in the reviewer's role.
```

Comment:
I think we should indicate where this field falls within a role definition so users can get the YAML right.